src/network/message_compact_blocks.rs

```diff
@@ -0,0 +1,44 @@
+//!
+//! BIP152  Compact Blocks network messages
+//!
+
+use util::bip152;
+
+/// sendcmpct message
+#[derive(PartialEq, Eq, Clone, Debug, Copy)]
```

Comment:
With what we are putting into our guidelines (and what is in Rust guidelines) we do derive `PartialOrd, Ord, Hash` opportunistically. The same applies to the structures below. And, if I am not wrong, some of them may be `Copy`.